### DISCUSSION TITLE
🚨 Project Update 🚨

### DISCUSSION THREAD

**Discussion by yuceltoluyag** [yuceltoluyag/Rise-of-Kingdoms-Bot#23] [Announcements]
---

# 🚨 Rise Of Kingdom Bot  🚨

Hi everyone,

I’m sad to announce that I’ll no longer be able to continue with the project. There are two main reasons for this:

1. 👶 **First**, I recently became a parent, and most of my time is now devoted to my new child.
2. 💻 **Second**, my current computer can no longer handle modern games and emulators. The FPS drops significantly, making it impossible for me to test the game properly. 

When I was recording videos for YouTube, I had to spend hours just to get 25-30 FPS, and even then, I was barely making it work. Back then, I could still test things despite occasional freezing. However, with recent updates to the game and emulators (I’ve tried all the emulators, even the oldest versions), the FPS drops drastically when processing graphics, sometimes even hitting zero. 🥶

![image](https://github.com/user-attachments/assets/518d5faa-70d3-4de0-a1cb-64f831b5ebbf)

I was really motivated to develop a brand-new GUI and add new features, but the testing process became extremely challenging. I didn’t want to share anything with you that I hadn’t thoroughly tested. 
![2024-07-30_23-53](https://github.com/user-attachments/assets/3a332e59-225a-49a9-b1ca-63de0fe26a66)

I’m not really into gaming PCs; my current computer is fine for my everyday tasks. But when it comes to processing graphics, it just doesn’t cut it. I had planned to share this update with you much earlier, but I’ve been so busy that I didn’t get the chance. 😅

I truly appreciate all the support and feedback you’ve given me. While I won’t be able to continue with the project right now, I’m grateful for having you all with me on this journey. 🙏

Best regards,  
[@yuceltoluyag ]

**Comment by serdarozkan41** [comment]
Eline sağlık kral, sağlıkla büyümesi dileğiyle.
